modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -438,6 +438,8 @@ def map_bgs_tracked_items(bgs_claim) # rubocop:disable Metrics/MethodLength
                              'Preparation for Notification',
                              'Complete'].include? claim_status
                            'ACCEPTED'
+                         elsif ['UNKNOWN'].include? claim_status # TODO: fill unknown
```

Comment:
Not sure if this should be a part of this commit since this will never run, but I feel like it should be documented somewhere